docs/TESTING.md

```diff
@@ -0,0 +1,27 @@
+# Testing
+
+## Testing Locally
+
+Run `shopify serve` to start the simulator, which runs your extension code locally. Navigate to the localhost url logged in your CLI to see the simulator. Here, you can see the changes you have made locally.
+
+### Adding mock data
+
+In the simulator, you can add mock data for each extension point.
+
+**Note:**  The shape of the data is different for each extension point.
+
+## Testing on a Shop
+
+Once your Argo Extension is ready, push your code to Shopify and publish a version to see your extension in a Shopify store.
```

Comment:
@ow is `Argo Extension` ok here? Or should we reword to match public docs?

Reply:
Maybe lose the Capital E?